### DISCUSSION TITLE
KODI works as aclient

### DISCUSSION THREAD

**Discussion by GWLBE** [GerryDazoo/Slinger#455] [General]
First, thanks to all that contributed to extending the life of our slingboxes.  I started with the original back in the day and now use a 350.  I've been using KODI as a client and works very well on my firetv, shield, PC, and raspberry Pi.  For those familiar with KODI, I go back to modifying the old XBMC, simply create a text file with your HTTP info.  Save it and rename txt to strm.  Put the Slingbox.strm file in one of the video shares of Kodi, navigate to it, open it the stream will open.

**Comment by GWLBE** [comment]
Also, for those with Directv and on their local network this web based remote works.  http://www.waltzremote.com/detect.html

**Reply by regmar505** [reply]
Thank you for sharing the info on how to use Slinger with Kodi. I tried on a Raspberry 3B+ and it works well though there is a slight delay with audio. Did you figure out how to use the remote control within Kodi?

**Comment by GWLBE** [comment]
I use the web based on remote.txt.  The link I provided I use when I remote into my PC where slinger is located.
